frontend/src/components/VMediaInfo/VRelatedMedia.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <VMediaCollection
+    v-if="showRelated"
+    :results="results"
+    :is-fetching="isFetching"
+    :collection-label="collectionLabel"
+    kind="related"
+    :related-to="relatedTo"
+    :search-term="searchTerm"
+    :aria-label="collectionLabel"
+  >
+    <template #header>
+      <h2
+        id="related-heading"
+        class="heading-6 mb-6"
+        :class="results.type === 'image' ? 'md:heading-5' : 'lg:heading-6'"
```

Comment:
This might not even be for the consistency with Figma. This is what we currently have on the live site, so I used it so that I don't have to also update the snapshots :)

Reply:
I opened a new issue for this to keep track: #3863